markdown/blog/neuroglia_project_interview.md

```diff
@@ -0,0 +1,82 @@
+
+In our conversation, Charles explained that Neuroglia's focus is mainly on orchestration. Unlike other players in their sector, Neuroglia doesn't take a "refactor or start-from-scratch" approach. Instead, they aim to integrate existing legacy systems into modern solutions, following an orchestration or event-driven approach.
+
+## The Origin of Neuroglia’s Journey with AsyncAPI
+
+Years ago, while working with orchestration tools, Neuroglia discovered Serverless Workflow, a CNCF project. They found the project while searching for orchestration tools that could standardize the creation and running of workflows for one of their clients. As Charles became deeply involved, he became the lead maintainer of the Serverless Workflow specification repository. Together with his associate, they also developed various SDKs, including Rust, .NET, and TypeScript.
+As their work on the Serverless Workflow project progressed, Charles encountered AsyncAPI. Recognizing its potential, he actively pushed for its support within the Serverless Workflow specification.
+At that time, the available .NET tooling options were limited and did not fulfill all their requirements, prompting Neuroglia to pursue a more vendor-neutral and flexible solution.
+
+## Building an AsyncAPI SDK from Scratch
+
+To solve their challenges, Neuroglia developed their own AsyncAPI SDK. They started with building domain models and then advanced to Fluent Builders, which allowed developers to create AsyncAPI documents using a Fluent code approach. This approach was followed by validation features implemented using Fluent Validation.
+The UI they developed initially resembled Swagger’s approach but later evolved to closely align with AsyncAPI’s official UI. Their solution has grown to include various modules, and now they are working towards integrating official HTML components for rendering documents.
+
+## Key Differentiators of Neuroglia’s Solution
+
+According to Charles, one of the key differentiators of Neuroglia’s AsyncAPI solution is their agnostic, out-of-the-box client, which supports most  of AsyncAPI specification bindings. This plug-and-play client allows developers to register services, specify the AsyncAPI document they want to connect to, and publish or subscribe to operations effortlessly.
+Another unique feature is Neuroglia’s support for concurrent AsyncAPI versions. Users can work with older documents and newer documents simultaneously without conflicts, enhancing the flexibility of their solution.
+
+## Problems Solved for AsyncAPI Users
+
+Neuroglia’s solution addresses several challenges for AsyncAPI users:
+
+- Version Compatibility: Hosting different versions of the AsyncAPI specification concurrently.
```

Comment:
[suggestion]: holding this and the other types of challenges presented in this list would make it more readable. 
```suggestion
- **Version Compatibility:** Hosting different versions of the AsyncAPI specification concurrently.
```